pkg/operator/target_config_reconciler.go

```diff
@@ -276,6 +276,14 @@ func (c *TargetConfigReconciler) manageConfigMap(descheduler *deschedulerv1.Kube
 		return nil, true, fmt.Errorf("enabling Descheduler LowNodeUtilization with Scheduler HighNodeUtilization may cause an eviction/scheduling hot loop")
 	}
 
+	// set PodLifetime if non-default
+	if descheduler.Spec.ProfileCustomizations.PodLifetime != nil {
```

Comment:
descheduler.Spec.ProfileCustomizations can be nil as well:
```
if descheduler.Spec.ProfileCustomizations != nil && descheduler.Spec.ProfileCustomizations.PodLifetime != nil {
```

Reply:
Good point, updated